### ISSUE TITLE
PM button not working

### ISSUE THREAD

**Issue by Dxvid64** [fizi/BS_Gaming_01#4]
![PM](https://user-images.githubusercontent.com/105501616/169084327-e473326f-353b-49e7-9f6d-9fbf771ef1af.png)
Nothing happens when clicking the PM button, (top right of the page), whilst using the BS_Gaming_01 theme.  Sometimes the button works for aprox. 1/2 hr. but most of the time it doesn't work at all. I have switched to the Bootstrap 3 theme where the PM button always works.  (I am using e107 2.3.2).

**Comment by Dxvid64** [comment]
No I'm not logged out, you just can't see the loggin button in the pic.

**Comment by Deltik** [comment]
I can't reproduce the issue:
![norepro](https://user-images.githubusercontent.com/1364268/170843464-8fb0ec73-8aa9-45ab-bfdd-ab0659ef106e.png)

My guess is a JavaScript evaluation error.  Check your browser's JavaScript console for more information.

**Comment by Dxvid64** [comment]
I disabled Java but it didn't make any difference unfortunately.

**Comment by Deltik** [comment]
We can only speculate unless you provide errors from your browser's JavaScript console.  Note that JavaScript is required for the dropdown to appear, so if there is an error interrupting JavaScript on the page, that would stop the dropdown from opening.

**Comment by Dxvid64** [comment]
[FB.txt](https://github.com/fizi/BS_Gaming_01/files/8804737/FB.txt) 
I think Firefox stopped supporting Java plugin a while ago so have attached output from browser console?  If that is not what you want let me know, in simple terms, and I will do it.
Thanks for your help.

**Comment by Deltik** [comment]
JavaScript, not Java.  On Firefox, JavaScript errors are logged in the message display pane of the Web Console.  Here is the documentation for the Web Console: https://firefox-source-docs.mozilla.org/devtools-user/web_console/

**Comment by Dxvid64** [comment]
Here is the web console output for the home page,

[FWC.txt](https://github.com/fizi/BS_Gaming_01/files/8813294/FWC.txt)

**Comment by Dxvid64** [comment]
Any luck?

**Comment by Deltik** [comment]
Those logs don't show any JavaScript errors.  Make sure the console is running when the page loads or reloads.

**Comment by Dxvid64** [comment]
Here is the Console output after removing Firefox data then reloading page with Console running,

[FC2.txt](https://github.com/fizi/BS_Gaming_01/files/8885612/FC2.txt)

**Comment by Deltik** [comment]
The attached log only shows the browser's internal logs, not the console log on the web page.

**Comment by Dxvid64** [comment]
Sorry, here it is,

[WC.txt](https://github.com/fizi/BS_Gaming_01/files/8886698/WC.txt) 

 and a version with all the little grey arrows expanded

[WCexpand.txt](https://github.com/fizi/BS_Gaming_01/files/8886699/WCexpand.txt)

**Comment by Deltik** [comment]
There aren't any JavaScript errors in the logs there, either.  Are you using the problematic theme and clicking on the button that won't open?

**Comment by Dxvid64** [comment]
Yes the login button on the home page.  The pm button doesn't work either.

**Comment by Dxvid64** [comment]
I'm currently using boostrap3 theme as the site is live, but switch to Bsgaming1 to do the tests you ask for.
